src/app/cluster/details/external-cluster/external-machine-deployment-list/template.html

```diff
@@ -53,28 +53,61 @@
         </td>
       </ng-container>
 
-      <ng-container matColumnDef="labels">
+      <ng-container *ngIf="cluster.cloud.aks"
```

Comment:
**Opinionated comment** if any concerns let me know.

Instead of adding individual *ngIf="cluster.cloud.aks"

create a wrapper <ng-container> and this condition there and move both Powe/Provsioning inside.

Reason:
- Avoid multiple checks 
- Tomorrow in case new props come will be under this check
- In case of refactoring or moving into sep component it will. be easy